### ISSUE TITLE
Class member with decorator is incorrectly marked unused

### ISSUE THREAD

**Issue by yseymour** [Microsoft/TypeScript#19826]
**TypeScript Version:**  2.6.1

**decorate.ts**
```ts
const f = (_p,_f) => {};
class foo {
	@f
	private bar():void {}
}
```

Compile with
```
tsc --experimentalDecorators --noUnusedLocals decorate.ts
```

**Expected behavior:**
Successful compile

**Actual behavior:**
Error:
```decorate.ts(4,10): error TS6133: 'bar' is declared but its value is never read.```
In a real world case, the side-effects of the decorator mean that bar() might in fact get called at some point (for instance, the decorator might attach an event listener) and so I believe this error should not be thrown.

**Comment by mhegazy** [comment]
Duplicate of https://github.com/Microsoft/TypeScript/issues/13120
